markets/bfp-market/test/integration/modules/MarginModule.debt.test.ts

```diff
@@ -151,6 +153,23 @@ describe('MarginModule Debt', async () => {
       );
     });
 
+    it('should revert when an order is present', async () => {
```

Comment:
I'd say should revert **paying debt** when an order is present

Reply:
It's part of the `payDebt` describe block so no need? Same name structure as all other reverts in this test suite.